### PR TITLE
Food list

### PR DESCRIPTION
**Pull request by Graycemuthui**
### Description
In this project I achieved the following:
- Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also 
   quantity of a given food).
- Should lead to a form that allows users to add new food.

### CODE REVIEW

**Review by myelin0 — CHANGES_REQUESTED** [review]
## Hi @Graycemuthui , 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

File: app/controllers/foods_controller.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    respond_to do |format|
+      if @food.update(food_params)
+        format.html { redirect_to food_url(@food), notice: "Food was successfully updated." }
+        format.json { render :show, status: :ok, location: @food }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```
Comment (myelin0) [inline app/controllers/foods_controller.rb:48]: hi @Graycemuthui! 👋  Good job implementing the CRUD methods, but the _**Update**_ method is not included as per project requirements [here](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md) . You can safely remove these lines. (you can also change line 2 after that if you prefer).

**Review by myelin0 — CHANGES_REQUESTED** [review]

File: app/controllers/foods_controller.rb (hunk 1 above)
Comment (Graycemuthui) [reply app/controllers/foods_controller.rb:48]: I don't think there's an issue if I implement them though its just additional knowledge

Comment (myelin0) [reply app/controllers/foods_controller.rb:48]: @Graycemuthui you are right, it is additional knowledge, but we have to follow the project`s requirements : 
**-A food list (with all CRUD implementation, except for 'update').**

Comment (myelin0) [reply app/controllers/foods_controller.rb:48]: Ok I see that you resolved the issue in the next branch , so I am closing this issue. Great job! 👍

**Review by myelin0 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Graycemuthui ,  

Your project is complete, awesome job for nailing it right 🥇 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Food List methods follow the CRUD.
- Foods page matches given wireframe.



<br>
Keep being awesome! 💻 🍷 🚀 
<br><br>

**Pull request merged**